Source/WebKit/Platform/cocoa/SharedMemoryCocoa.cpp

```diff
@@ -273,4 +273,32 @@ WTF::MachSendRight SharedMemory::createSendRight(Protection protection) const
     return makeMemoryEntry(m_size, toVMAddress(m_data), protection, MACH_PORT_NULL);
 }
 
+void SharedMemory::setVolatile()
+{
+    ASSERT(m_data);
+
+    int purgeableState = VM_PURGABLE_VOLATILE;
+    auto kr = mach_vm_purgable_control(mach_task_self(), toVMAddress(m_data), VM_PURGABLE_SET_STATE, &purgeableState);
+
+    RELEASE_LOG_ERROR_IF(kr != KERN_SUCCESS, VirtualMemory, "SharedMemory::setVolatile: Failed to set memory volatile. Error: %" PUBLIC_LOG_STRING " (%x)", mach_error_string(kr), kr);
```

Comment:
Why do not we return whether the mach call has succeeded or not like what we do in `SharedMemory::setNonVolatile()`?